integrations/drip/drip_test.go

```diff
@@ -0,0 +1,177 @@
+package drip
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/jipiboily/forwardlytics/integrations"
+)
+
+func TestIdentifyErrorWhenNoEmail(t *testing.T) {
+	drip := Drip{}
+	identification := integrations.Identification{
+		UserID:     "123",
+		UserTraits: map[string]interface{}{},
+	}
+	err := drip.Identify(identification)
+	if err == nil {
```

Comment:
Maybe we can add an issue to fix this and I can take a look after the other stuff is done? Might need to do it for all integrations I guess?